cache/refs.go

```diff
@@ -1232,7 +1291,8 @@ func (sr *immutableRef) unlazyLayer(ctx context.Context, dhs DescHandlers, pg pr
 
 	key := fmt.Sprintf("extract-%s %s", identity.NewID(), sr.getChainID())
 
-	err = sr.cm.Snapshotter.Prepare(ctx, key, parentID)
+	opts := snapshots.WithLabels(map[string]string{"containerd.io/snapshot.ref": string(desc.Digest)})
```

Comment:
Should not be enabled by default.

Reply:
Fixed, now this label is only for overlaybd.